microsoft/electionguard-cpp
# Feature/bench win

_🚨 Please review the [guidelines for contributing](../CONTRIBUTING.md) to this repository._

### Issue

Fixes #189

### Description
also a partial implementation of #14 

Adds an `ElectionGuard.Bench` project to the .net framework solution and adds boilerplate infrastructure to asynchronously encrypt ballot selections.  This PR demonstrates a single benchmark test case that encrypts a ballot.

### Testing
on windows:
1. make build-msvc
2. open the .net framework solution and build the project.
3. run the `ElectionGuard.Bench` project from the command line (from the /bin folder)
4. expect: it outputs the time it takes to run

on other platforms:
1. run `make test` and `make bench`
